tests/phpunit/diagnostics/class-wc-stripe-diagnostics-cleanup-test.php

```
@@ -0,0 +1,103 @@
+	 * @return void
+	 */
+	public function test_daily_cleanup_keeps_trace_exactly_at_edge() {
+		$this->store->create( 'edge' );
+		// 7 days minus a buffer: still within window.
+		$this->set_times( 'edge', time() - ( 7 * DAY_IN_SECONDS - 60 ), time() - HOUR_IN_SECONDS );
+
+		$deleted = $this->cleanup->run_daily_cleanup();
+
+		$this->assertSame( 0, $deleted );
+		$this->assertNotNull( $this->store->get( 'edge' ) );
+	}
```

PHPMD (2.15.0)

[error] The method test_daily_cleanup_keeps_trace_exactly_at_edge is not named in camelCase. (undefined)

(CamelCaseMethodName)